src/commands/Admin/update.ts

```diff
@@ -1,47 +1,43 @@
-import type { Message } from '@klasa/core';
-import { AlestraCommand, AlestraCommandOptions } from '@lib/structures/AlestraCommand';
-import { PermissionLevels } from '@lib/types/Enums';
+import { ApplyOptions } from '@sapphire/decorators';
+import { Args, Command, CommandOptions } from '@sapphire/framework';
 import { codeBlock, cutText } from '@sapphire/utilities';
-import { ApplyOptions } from '@skyra/decorators';
 import { Emojis, rootFolder } from '@utils/constants';
 import { exec } from '@utils/exec';
 import { sleep } from '@utils/sleep';
-import { remove } from 'fs-nextra';
+import type { Message } from 'discord.js';
+import { rm } from 'fs/promises';
 import { resolve } from 'path';
 
-@ApplyOptions<AlestraCommandOptions>({
+@ApplyOptions<CommandOptions>({
 	aliases: ['pull'],
 	description: 'Update the bot',
-	guarded: true,
-	permissionLevel: PermissionLevels.BotOwner,
-	usage: '[branch:string]'
+	preconditions: ['OwnerOnly'],
+	strategyOptions: { flags: ['fullRebuild'] }
 })
-export default class extends AlestraCommand {
-	public async run(message: Message, [branch = 'main']: [string?]) {
+export default class UserCommand extends Command {
+	public async run(message: Message, args: Args) {
+		const branch = await args.pick('string').catch(() => 'main');
 		await this.fetch(message, branch);
 		await this.updateDependencies(message);
-		await this.cleanDist(message);
+		if (args.getFlags('fullRebuild')) await this.cleanDist(message);
```

Comment:
```suggestion
		if (args.getFlags('clean')) await this.cleanDist(message);
```

Lets match Skyra's flags